trace/Sampling.md

```diff
@@ -12,7 +12,11 @@ The Sampling bit is always set only at the start of a Span, using a `Sampler`
 * `AlwaysSample` - sampler that makes a "yes" decision every time.
 * `NeverSample` - sampler that makes a "no" decision every time.
 * `Probability` - sampler that tries to uniformly sample traces with a given probability. When 
-applied to a child `Span` of a **sampled** parent `Span`, the child `Span` keeps the sampling decision.
+applied to a child `Span` of a **sampled** parent `Span`, the child `Span` keeps the sampling
+decision.
+* `QPS` - sampler that tries to sample traces based on QPS (e.g. 1 trace/second). When applied to a
```

Comment:
@nevinheintze mentioned offline that we might want to set the default rate significantly lower than 1 trace/sec to avoid issues when there are a very large number of tasks - maybe 0.1 trace/sec.

Reply:
Agreed.